projects/1.20/assets/mekanism-lasers/mekanism_lasers/lang/zh_cn.json

```
@@ -0,0 +1,33 @@
+{
+  "block.mekanism_lasers.laser_stopper": "激光阻挡器",
+
+  "block.mekanism_lasers.basic_laser": "基础激光器",
+  "block.mekanism_lasers.basic_toggleable_laser": "基础可切换激光器",
```

```suggestion
  "block.mekanism_lasers.basic_laser": "基础激光",
  "block.mekanism_lasers.basic_toggleable_laser": "基础可开关激光",
```
Mek1.20本体用的是`激光`，这里的激光器和本体的有无区别
toggleable建议修改